packages/multi-column/src/test/commands/__snapshots__/textCursorPosition.test.ts.snap

```diff
@@ -0,0 +1,273 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`Test getTextCursorPosition & setTextCursorPosition > Column 1`] = `
+{
+  "block": {
+    "children": [],
+    "content": [
+      {
+        "styles": {},
+        "text": "Column Paragraph 0",
+        "type": "text",
+      },
+    ],
+    "id": "column-paragraph-0",
+    "props": {
+      "backgroundColor": "default",
+      "textAlignment": "left",
+      "textColor": "default",
+    },
+    "type": "paragraph",
+  },
+  "nextBlock": {
+    "children": [],
+    "content": [
+      {
+        "styles": {},
+        "text": "Column Paragraph 1",
+        "type": "text",
+      },
+    ],
+    "id": "column-paragraph-1",
+    "props": {
+      "backgroundColor": "default",
+      "textAlignment": "left",
+      "textColor": "default",
+    },
+    "type": "paragraph",
+  },
+  "parentBlock": {
```

Comment:
Parent block is a column list for both snapshots - I think this is ok but technically it should maybe be the column since that also counts as a block? @YousefED

Reply:
yes, should be the column, nice spot